numba/tests/test_parallel_backend.py

```diff
@@ -536,7 +536,7 @@ def each_env_var(self, env_var: str):
         env['NUMBA_THREADING_LAYER'] = 'default'
         env['NUMBA_THREADING_LAYER_PRIORITY'] = env_var
 
-        code = """import sys
```

Comment:
Perhaps use `dedent` for improved readability? 

https://docs.python.org/3/library/textwrap.html#textwrap.dedent

Reply:
7363275